pages/contactus.tsx

```diff
@@ -9,24 +9,68 @@ import TextField from "snet-ui/TextField";
 const categories = ["Airdrop Enquiry"];
 
 export default function ContactUs() {
+  const [message, setMessage] = useState("");
+  const [email, setEmail] = useState("");
+
+  const [error, setError] = useState({ email: "", message: "" });
   const [category, setCategory] = useState("Airdrop Enquiry");
 
   const handleChange = (event: SelectChangeEvent) => {
     setCategory(event.target.value as string);
   };
+  const validateEmail = () => {
+    setEmail(event.target.value);
+  };
+  const validateMessage = () => {
+    setMessage(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    setError({});
+
+    if (message === "") {
+      setError((prevError) => ({
+        ...prevError,
+        message: "Message should not be empty",
+      }));
+    }
+    const reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+    if (!email) {
+      setError((prevError) => ({
+        ...prevError,
+        email: "Enter the valid email",
+      }));
+    }
+  };
 
   return (
     <CommonLayout>
       <Box p={5}>
         <Typography align="center" color="primary" variant="h3">
           Contact Us
         </Typography>
+
         <Grid container sx={{ my: 3 }} spacing={2}>
           <Grid item xs={12} sm={6}>
-            <TextField color="primary" label="Your Name (Optional)" placeholder="Hello World" fullWidth />
+            <TextField
+              color="primary"
+              required
+              label="Your Name (Optional)"
+              placeholder="Hello World"
```

Comment:
`Hello world` is not an appropriate placeholder. 
This is okay for local testing but you have to use proper / meaningful values before committing the code.